### PR TITLE
update: OS snapshot migration feedback

### PR DESCRIPTION
**Pull request by harshini-rangaswamy**
## Describe your changes

Updated OS snapshot migration documentation based on feedback.
 - Included corrections to API endpoints and CLI commands and additional clarity on the reapplication of ISM policy.
 - Included other feedback based on emulating a customer setup

## Checklist

- [x] The first paragraph of the page is on one line.
- [x] The other lines have a line break at 90 characters.
- [x] I checked the output.
- [x] I applied the [style guide](styleguide.md).
- [x] My links start with `/docs/`.

### CODE REVIEW

File: docs/products/opensearch/howto/migrate-ism-policies.md [hunk 1]
```diff
@@ -67,6 +76,11 @@ To reapply ISM policies to indices in Aiven for OpenSearch:
    python avn-re-apply-ism-policies.py --config path-to-config-file
    ```
 
+## Re-running the ISM script
+
+You can rerun the ISM script if needed. Use the --force option to bypass the check
+that prevents it from running more than once.
```
Comment (hoang-vo) [inline docs/products/opensearch/howto/migrate-ism-policies.md:82]: We should advise to only run this one time after you have finished all the data migration

File: docs/products/opensearch/howto/migrate-snapshot-data-opensearch.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ limited: true
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-Aiven for OpenSearch enables you to restore data from external OpenSearch or Elasticsearch snapshots, enabling seamless migration from third-party repositories.
+Aiven for OpenSearch lets you to restore data from external OpenSearch or Elasticsearch snapshots, enabling seamless migration from third-party repositories.
```
Comment (hoang-vo) [inline docs/products/opensearch/howto/migrate-snapshot-data-opensearch.md:10]: seamless in our context might means different things, after a couple of bugs we are observing, im not sure if we can say it is going to be seamless

File: docs/products/opensearch/howto/migrate-ism-policies.md (hunk 1 above)
Comment (harshini-rangaswamy) [reply docs/products/opensearch/howto/migrate-ism-policies.md:82]: added a note to this section.

File: docs/products/opensearch/howto/migrate-snapshot-data-opensearch.md (hunk 2 above)
Comment (harshini-rangaswamy) [reply docs/products/opensearch/howto/migrate-snapshot-data-opensearch.md]: removed this word

**Review by hoang-vo — APPROVED** [review]

File: docs/products/opensearch/howto/migrate-snapshot-data-opensearch.md [hunk 3]
```diff
@@ -20,25 +16,27 @@ Aiven for OpenSearch supports restoring snapshots from the following cloud provi
...
 ## Prerequisites
 
 Before you begin, ensure that:
 
 - You have an active Aiven account with a configured project.
 - A target Aiven for OpenSearch service within your project.
-- Your OpenSearch or Elasticsearch snapshot is from version 7.10.2 or earlier and is
-  compatible with your target service.
+- Your OpenSearch or Elasticsearch snapshot is from version 7.10.2 or earlier, or
+  from OpenSearch 1.x/2.x, and is compatible with your target service.
```
Comment (hoang-vo) [inline docs/products/opensearch/howto/migrate-snapshot-data-opensearch.md:29]: and not newer than target OS

**Review by hoang-vo — CHANGES_REQUESTED** [review]

File: docs/products/opensearch/howto/migrate-snapshot-data-opensearch.md (hunk 3 above)
Comment (harshini-rangaswamy) [reply docs/products/opensearch/howto/migrate-snapshot-data-opensearch.md]: updated

**Review by ArthurFlag — APPROVED** [review]

**Review by hoang-vo — APPROVED** [review]

**Pull request merged**